jetbrains-core/resources/META-INF/plugin.xml

```diff
@@ -30,33 +30,26 @@
     If you come across bugs with the toolkit or have feature requests, please raise an <a
         href="https://github.com/aws/aws-toolkit-jetbrains/issues">issue</a>.
 </p>
-
```

Comment:
Nit - let's make sure we don't do just whitespace changes unless it's park of an overall reformatting effort.